### PR TITLE
Unhardcode nuke and antinukes from the antinuke gadget/widget

### PR DESCRIPTION
**Pull request by Mach565**
Not sure how this was missed in the initial unhardcoding, but here it is.

### CODE REVIEW

File: units/staticnuke.lua [hunk 1]
```diff
@@ -21,6 +21,7 @@ return { staticnuke = {
     stockpiletime  = [[180]],
     stockpilecost  = [[3240]],
     priority_misc  = 1, -- Medium
+    isNuke = true,
```
Comment (sprunk) [inline units/staticnuke.lua:24]: I am scared of uppercase.

**Review by sprunk — CHANGES_REQUESTED** [review]
`is_nuke`, also missing the UI part.

**Comment by Mach565** [comment]
> `is_nuke`, also missing the UI part.

UI part?

**Comment by sprunk** [comment]
> UI part?

The widget that draws the green and red lines and circles when aiming a nuke.

**Comment by sprunk** [comment]
Looks blind and untested.

**Pull request merged**